salvador/Helpers/FetchImage.swift

```diff
@@ -7,11 +7,11 @@
 
 import SwiftUI
 
+
 enum ImageError: Error {
-    case badURL
+    case badURL, missingParameters
 }
 
-
 class CreateImage {
```

Comment:
I would give file and corresponding class type the same name. And maybe you could get a bit more descriptive about what `CreateImage` is. Is it a view model, a view, a service, ...? I would suggest it is a service named `CreateImageService`